### ISSUE TITLE
Translation timeout (I will post the srt file)

### ISSUE THREAD

**Issue by toprak** [sinedie/SRTranslator#43]
using 2.3.0 here is the [srt](https://we.tl/t-d5M2J1Iwgm)
```INFO:root:Getting a new Proxy from https://www.sslproxies.org/
INFO:root:Creating Selenium Webdriver instance
INFO:root:Going to https://www.deepl.com/translator
Loading Anime/[Judas] Horimiya (Season 1) [1080p][HEVC x265 10bit][Multi-Subs]/[Judas] Horimiya - S01E07.srt
... Translating chunk. 0 %
... Translating chunk. 24 %
... Translating chunk. 48 %
Traceback (most recent call last):
  File "/usr/lib/python3.10/runpy.py", line 196, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.10/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/tbb/.local/lib/python3.10/site-packages/srtranslator/__main__.py", line 110, in <module>
    srt.translate(translator, args.src_lang, args.dest_lang)
  File "/home/tbb/.local/lib/python3.10/site-packages/srtranslator/srt_file.py", line 151, in translate
    translation = translator.translate(
  File "/home/tbb/.local/lib/python3.10/site-packages/srtranslator/translators/deepl_scrap.py", line 133, in translate
    raise TimeOutException("Translation timed out")
srtranslator.translators.base.TimeOutException: Translation timed out
Traceback (most recent call last):
  File "/home/tbb/translate.py", line 104, in <module>
    main(args)
  File "/home/tbb/translate.py", line 24, in main
    translate_srt(srt_file)
  File "/home/tbb/translate.py", line 58, in translate_srt
    subprocess.run(cmd, shell=True, check=True)
  File "/usr/lib/python3.10/subprocess.py", line 526, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command 'python -m srtranslator 'Anime/[Judas] Horimiya (Season 1) [1080p][HEVC x265 10bit][Multi-Subs]/[Judas] Horimiya - S01E07.srt' -i en -o tr -v -t deepl-scrap' returned non-zero exit status 1.```

**Comment by sinedie** [comment]
Can't replicate. Maybe try run it with -S flag and show me what happens, cause It works for me.

**Comment by sinedie** [comment]
Translating to spanish is not problem for me. Please run it with ```-s``` flag to be sure.

**Comment by toprak** [comment]
I run it with it already though @sinedie translation was eng to tr

**Comment by sinedie** [comment]
Ok... thats really weird, but I'll try in another resolutions so I can replicate. Ty for invest time finding a solution

**Comment by toprak** [comment]
adding ` self.driver.execute_script("document.body.style.zoom='80%'") ` to `def __init__(self, driver: Optional[WebDriver] = None):` in deepl_scrap.py solved my problem.

**Comment by toprak** [comment]
it started again, I guess it because of selected proxy for that session, i guess we should select the proxies by location I mean which is closer to user @sinedie

**Comment by sinedie** [comment]
DeepL page has different version per country... We can do that, but I don't want to mess with getting the current user location, it would be better if you can choose the enabled countries for the proxy.

Also don't want to force the selection of the country, so it could be a custom usage, not built-in in the core, but shown in examples how to do it. Just to keep the core as simple as possible (and also It is a custom configuration, so...)

It could work something like this:
```
   proxy = create_proxy(country_id=['US']) # the country ids are the ones in https://www.sslproxies.org/
   driver = create_driver(proxy)
   translator = DeeplTranslator(driver)
```

In CLI usage... maybe enable add your own proxy url? (not sure about that) or even better, always work with US proxies, that way it's gonna be the same page version on each run

**Comment by toprak** [comment]
@sinedie Friend sorry I know i am bothering you but I guess I really found the solution right now!
srtranslator was stuck the lines like this `<i></i><i>Frankfurter</i>` I delete these coding, and ocnvert them for example Frankfurter it worked, I think srtranslator should handle these cases

**Comment by toprak** [comment]
And tried again, without editing it does not work, I will provide edited and non edited version of srts to you.
https://send.vis.ee/download/d3079522309a4386/#HL3pAC-d-DIyqlVMNI5Mvg

**Comment by sinedie** [comment]
No problem. But that is not the error. Actually, the first subtitle is ```<i></i><i></i>```. This library indeed handle this cases, and convert the line to an empty string. So.... when deepl translate it the translation length (number of lines) is different from source text length, that's why it timesout : one of the checks to know if translation is done is to check the equalty of those lenghts.

A possible quick solution could be to replace empty string for a ```...``` character. As I really don't want to mess up deleting subtitles lines. As this library is not meant for that.

Gonna work on it. You could just add it by hand by now and translate it. Ty for catching this. 👍

**Comment by sinedie** [comment]
@toprak Solved in v0.2.5 already in [PyPi](https://pypi.org/project/srtranslator/0.2.5/)

**Comment by toprak** [comment]
then we can close this, thanks!
